src/main/webapp/index.html

```diff
@@ -89,12 +89,18 @@ <h3>Taiwan Team 8 Project: Restaurant Recommendation</h3>
                 <button id="results" class="btn btn-outline-info"><i class="lni lni-restaurant"></i> Get Recommendation</button>
 
                 <!-- Result -->
-                <div class="card result-card">
-                    <i class="lni lni-restaurant card-img-top"></i>
+                <div id="result-restaurant" class="card result-card" style="visibility: hidden;">
+                    <h6 class="card-title">Recommended Restaurant:</h6>
+                    <img id="icon" class="card-img-top restaurant-photo" src="/images/resta.png" >
                     <!-- <img class="card-img-top" src="..." alt="Card image cap"> -->
                     <div class="card-body">
                         <h5 class="card-title" id="places">What's Your Next Restaurant?</h5>
-                        <p class="card-text" id="placeDetails">Restaurant Details (Address, Contact, Cuisine Type, Price Level, Dine In/Takeout, Max Cap etc)</p>
+                        <p class="card-text" id="address"></p>
+                        <p class="card-text" id="phone"></p>
+                        <p class="card-text" id="website"></p>
+                        <p class="card-text" id="rating"></p>
+
+                        <!-- <p class="card-text" id="placeDetails">Restaurant Details (Address, Contact, Cuisine Type, Price Level, Dine In/Takeout, Max Cap etc)</p> -->
                         <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
                     </div>
                 </div>
```

Comment:
if no results are found (after having found results before), the sidebar doesn't seem to clear out automatically (see below):

![Screen Shot 2020-08-30 at 6 36 21 PM](https://user-images.githubusercontent.com/31350668/91656982-f9f6b600-eaef-11ea-812e-aaf2f3977676.png)

Reply:
i realize this inadvertently reveals where i live lol, but lets ignore that

Reply:
when no results are found (first time getting recommendation), the sidebar doesn't show anything so user might be confused (see below):

![Screen Shot 2020-08-30 at 6 40 47 PM](https://user-images.githubusercontent.com/31350668/91657028-6e315980-eaf0-11ea-922c-441128aecbd5.png)

could be a good idea to give user feedback like "sorry try another search"

Reply:
- when first getting started, will not show any results. 
understand the bug when there's no restaurant shown which happens in backend, such that place.name didnt get updated at all... hmm am fixing this one now!

Reply:
there should probably be some sort of change if there's no results with the first use? like if a user wanted to find a $$$$ restaurant in a remote place, there should be some indicator that the search has completed and they need to widen their search